dev-python/flask-paginate/flask-paginate-0.7.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz -> ${P}-${PV}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-python/flask[${PYTHON_USEDEP}]"
+BDEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? ( dev-python/pytest[${PYTHON_USEDEP}] )
+"
+
+python_test() {
+	pytest -vv tests/tests.py || die "tests failed with ${EPYTHON}"
+}
```

Comment:
You can replace all of these lines with a single `distutils_enable_tests pytest`